waspc/data/Generator/templates/react-app/src/queries/_query.js

```diff
@@ -2,14 +2,14 @@
 import { callOperation } from '../operations'
 import * as resources from '../operations/resources'
 
-const {= queryFnName =} = async (args) => {
-  return await callOperation('{= queryRoute =}', args)
+function {= queryFnName =}(args) {
```

Comment:
Just to check:
1. Does this completely resolve #487 ? I am asking because it got closed by this PR. If not, you will want to reopen it.
2. Does this change any of the interface toward Wasp users? If so, we should update the docs / examples. But I don't think it does, right?

Reply:
> Does this completely resolve https://github.com/wasp-lang/wasp/issues/487 ? I am asking because it got closed by this PR. If not, you will want to reopen it.

Thanks, will do

>  Once we have types (TypeScript) those will be documentation.

Linters often complain about redundant `async` and `await`. But yes, let's hope TypeScript makes this easier.